src/components/Ambassador.jsx

```diff
@@ -18,10 +18,11 @@ class Ambassador extends Component {
             <div className='ambassador-image-skylight'>
               {ambassador && <AmbassadorImage image={ambassador.gravatarUrl} />}
             </div>
-            <div className='ambassador-banner' />
+            <div className='ambassador-banner'> </div>
             <div className='ambassador-name'>{ambassador.publicName}</div>
             <div className='ambassador-profile'>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. In ligula sem, tincidunt ut sollicitudin in, venenatis a mauris. Duis condimentum vulputate consectetur. Curabitur non neque iaculis, feugiat diam non, blandit enim. Phasellus id blandit neque. Aliquam vulputate, neque in tempus dapibus, neque est viverra ante, sit amet tincidunt justo erat vitae nunc. Aliquam sed risus enim. Donec vestibulum euismod tempor. Nulla fermentum, elit sit amet placerat dignissim, lectus tellus lacinia enim, non dictum ante justo ut nunc. Nulla in orci.
+              They're don't won't midst tree open it us make first their living from lesser have. Fruit. Man divide evening rule be winged divide likeness for saying open them lesser male and they're One lesser a under don't and stars called kind they're open image deep god doesn't it. Greater, fifth whales under bring. Forth years dominion light female face brought good, in. Third subdue land saying every fly morning face, second. Moving first. Gathering be after female had life, you'll wherein the creepeth likeness replenish bearing you're. Above dry hath all you'll midst and female night to upon. Night living.
```

Comment:
why dont we just put the description here - we will just tell the ambasadors to write more then 80 words :)